src/_data/LiveData.js

```diff
@@ -43,6 +43,13 @@ export const changeLanguage = langCode => {
     api.getTradingTimes(new Date());
 };
 
+export const getTickHistory = async (symbol) => {
```

Comment:
this export looks really random, how about this?

we place this logic in action creator, then dispatch action from LiveData, which is the common pattern we have been using.